docs/api.md

```diff
@@ -194,6 +194,43 @@ from pettingzoo.utils import random_demo
 random_demo(env, render=True, episodes=1)
 ```
 
+### Playing Alongside Trained Policies
+
+Sometimes, you may want to control a singular agent and let the other agents be controlled by trained policies.
+Some games support this via:
+
+``` python
+import time
+from pettingzoo.butterfly import knights_archers_zombies_v8
+
+env = knights_archers_zombies_v8.env()
+env.reset()
+
+manual_policy = knights_archers_zombies_v8.ManualPolicy(env)
+
+for agent in env.agent_iter():
+    observation, reward, done, info = env.last()
+
+    if agent == manual_policy.agent:
+        action = manual_policy(observation, agent)
+    else:
+        action = policy(observation, agent)
```

Comment:
Might want to mention that this `policy` is a function you need to define yourself, it comes from something you trained or wrote, not from anywhere in pettingzoo. This has caused confusion elsewhere in the documentation.

Reply:
Gotcha